synapse/lib/stormlib/iters.py

```diff
@@ -0,0 +1,33 @@
+import synapse.lib.stormtypes as s_stormtypes
+
+@s_stormtypes.registry.registerLib
+class LibIters(s_stormtypes.Lib):
+
+    _storm_lib_path = ('iters', )
+    _storm_locals = (
+        {
+            'name': 'enum', 'desc': 'Yield (<indx>, <item>) tuples from an iterable or generator.',
+             'type': {
+                'type': 'function', '_funcname': 'enum',
+                'args': (
+                    {'type': 'iter', 'name': 'genr', 'desc': 'An iterable or generator.'},
```

Comment:
Do we want to add a `indx` argument ( default to 0 ) to start the index at? For some use cases being able to start indexing at `1` could be nice.

Reply:
I'll leave that for a future-self when it becomes needed :)